components/pkg-export-kubernetes/src/cli.rs

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) 2017 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use clap::{App, Arg};
+use std::result;
+
+use export_docker as docker;
+
+/// A Kubernetes-specific clap:App wrapper
+#[derive(Clone)]
+pub struct Cli<'a, 'b>
+where
+    'a: 'b,
+{
+    pub app: App<'a, 'b>,
+}
+
+impl<'a, 'b> Cli<'a, 'b> {
+    pub fn new(name: &str, about: &'a str) -> Self {
+        let app = docker::Cli::new(name, about).app;
+
+        Cli { app: app }
+    }
+
+    pub fn add_all_args(self) -> Self {
+        self.add_docker_args()
+            .add_output_args()
+            .add_runtime_args()
+            .add_secret_names_args()
+            .add_bind_args()
+    }
+
+    pub fn add_docker_args(self) -> Self {
+        let cli = docker::Cli { app: self.app };
+        let app = cli.add_base_packages_args()
+            .add_builder_args()
+            .add_tagging_args()
+            .add_publishing_args()
+            .add_pkg_ident_arg(docker::PkgIdentArgOptions { multiple: true })
```

Comment:
I thought this was supposed to be false here?

Reply:
ah yes, good catch.